lib/asset_symlink.rb

```diff
@@ -27,16 +27,25 @@ def self.execute config
 
   def self.normalize_configuration config
     case config
-    when Hash 
+    when Hash
       config
     when String
       {config => config}
-    when Array 
+    when Array
       config.inject({}) do |result, element|
         result.merge(normalize_configuration(element))
       end
     when NilClass
```

Comment:
Maybe more explicit if this is triggered by doing something like `config.asset_symlink = :all ` ?